packages/smooth_app/pubspec.lock

```diff
@@ -492,7 +492,7 @@ packages:
       name: js
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.6.4"
+    version: "0.6.3"
```

Comment:
How come you get lower version numbers? I suggest that you refresh something like `flutter pub get`